script/Main.lua

```diff
@@ -31,6 +31,10 @@ function InitSystem()
         GlobalRestrict.On()
 
         dofile('./script/Config/Version.lua')
+        if Config.gameVersion ~= ffi.string(Engine.GetVersion()) then
```

Comment:
Can we somehow do this without `ffi.string()`

Reply:
https://github.com/Limit-Theory-Redux/ltheory/pull/191